### PR TITLE
[#2011] Remove plan/tier feature references

### PR DESCRIPTION
**Pull request by tangrammer**
- [ ] **Update release notes if necessary**

Relates #2011

### CODE REVIEW

File: backend/resources/akvo/lumen/migrations/tenant_manager/004-remove-plan.up.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DROP INDEX plan_tenant_ends_key;
+ALTER TABLE plan DROP CONSTRAINT plan_pkey;
+DROP TABLE IF EXISTS plan;
```
Comment (iperdomo) [inline backend/resources/akvo/lumen/migrations/tenant_manager/004-remove-plan.up.sql]: `DROP TABLE` support `CASCADE`, so you don't need to delete child objects manually.
https://www.postgresql.org/docs/11/sql-droptable.html

Comment (tangrammer) [reply backend/resources/akvo/lumen/migrations/tenant_manager/004-remove-plan.up.sql]: keeping just 
```
DROP TABLE IF EXISTS plan;
DROP TYPE IF EXISTS tier;


```

**Comment by tangrammer** [comment]
@iperdomo removed the migration file changes

**Comment by iperdomo** [comment]
@tangrammer there are some SQL changes there, I would suggest that we move them to another PR

**Comment by iperdomo** [comment]
Forget my last comment :man_facepalming: ... those sql changes are HugSQL related

**Comment by tangrammer** [comment]
ready again after this comment https://github.com/akvo/akvo-lumen/pull/2614#issuecomment-600685708

**Pull request merged**